src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -1064,6 +1067,12 @@ private void SetAffinity(Process process, int cores, bool addPerformanceHint)
             try
             {
                 var currentlyAssignedCores = Bits.NumberOfSetBits(process.ProcessorAffinity.ToInt64());
+                if (IgnoreProcessorAffinityChanges && currentlyAssignedCores != cores)
+                {
+                    Console.WriteLine($"Processor affinity change detected. Current cores: {currentlyAssignedCores}. Requested cores: {cores}.");
```

Comment:
Why do we want to run `Console.WriteLine` here?

Reply:
It is only used in tests. I can easily check the test output because of that.

Reply:
It is only used in tests right now, I worry about this changing down the line.

Reply:
After https://issues.hibernatingrhinos.com/issue/RavenDB-14605 is done I will switch this to throw during test execution.

Reply:
Great!